src/openapi.rs

```diff
@@ -50,3 +50,17 @@ pub struct OpenAPI {
     #[serde(flatten, deserialize_with = "crate::util::deserialize_extensions")]
     pub extensions: IndexMap<String, serde_json::Value>,
 }
+
+impl OpenAPI {
+    /// Iterates through all [Operation]s in this API.
+    ///
+    /// The iterated items are pairs of `(&str, &Operation)` containing the path and the operation.
+    ///
+    /// Path items containing `$ref`s are skipped.
+    pub fn operations(&self) -> impl Iterator<Item = (&str, &Operation)> {
```

Comment:
if we add the operation type to Operation::iter below I'd suggest we include it here as well so the tuple would be (path, op_type, Operation)